uploadNeuroDB/tarchive_validation.pl

```diff
@@ -313,20 +313,6 @@ =head2 Methods
     ##it can be inserted per minc into the MRICandidateErrors
 }
 
-################################################################
-### Extract the tarchive and feed the dicom data dir to ######## 
-### The uploader ###############################################
-################################################################
-my ($ExtractSuffix,$study_dir,$header) = 
```

Comment:
Is wondering what the variable `$ExtractSuffix` and `$header` was doing here before? The `$study_dir` was used in below, but not these two.

Reply:
It's there because the function used to extract and parse the tarchive returns 3 values after completion. Those two variables are not useful in this case (as the extraction, lol).

Reply:
I suppose so as well, but wanted to double check. It looks great to me.